docs/tutorials/how-to-edit-configmap.md

```diff
@@ -0,0 +1,164 @@
+# How to Edit ConfigMap 
```

Comment:
could we expand this to "How to create and edit a ConfigMap"  and add steps to create a new ConfigMap to the tutorial? Or would that be a "How to create new resources" tutorial?

Reply:
Please provide steps to create new resources.

Reply:
Done